.github/workflows/rust.yml

```
@@ -63,6 +50,29 @@ jobs:
         if: matrix.rust == '1.29.0'
         run: cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose
       - name: Running cargo
-        env: ${{ matrix.env }}
+        env:
+          DO_FEATURE_MATRIX: true
         run: ./contrib/test.sh
 
+  WASM:
+    name: WASM
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [stable, beta, nightly] # WASM requires Rust 1.30
+    steps:
+      - name: Checkout Crate
+        uses: actions/checkout@v2
+      - name: Install clang
+        run: sudo apt-get install -y clang-9
+      - name: Checkout Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

```suggestion
          toolchain: ${{ matrix.rust }}
```

Need to actually use the matrix variable :)

---

Nice reviewing, thanks man. Will re-spin tomorrow.